mpop_kiosk/WidgetPreviousNext.qml

```diff
@@ -1,25 +1,61 @@
 import QtQuick 2.0
 import QtQuick.Layouts 1.0
 import QtQuick.Controls 2.0
+import QtQuick.Controls 2.15
 
-RowLayout {
+ColumnLayout {
     id: thisWidget
+    Layout.topMargin : 300
 
     signal previousButtonClicked()
     signal nextButtonClicked()
 
     // TODO: Improve the look of these buttons
-    Button {
-        text: "<-"
+
+    RoundButton {
+        id: button
+        //text: "\u2191"
+        icon.source: "Downloads/uparrow.svg"
+        Layout.fillHeight: true
+        Layout.fillWidth: true
+        Layout.maximumWidth: 80
+        Layout.maximumHeight: 80
+
         onClicked: {
             thisWidget.previousButtonClicked();
         }
+
     }
 
-    Button {
-        text: "->"
+    Label {
+        id: label1
+        text: qsTr("PRECEDENTE")
```

Comment:
Use "Previous" As a text a capitalize it

Reply:
i did it sir